client/data/emails/use-get-titan-mailbox-availability.js

```diff
@@ -1,6 +1,14 @@
 import { useQuery } from 'react-query';
 import wpcom from 'calypso/lib/wp';
 
+export const getCacheKey = ( domain, mailbox ) => [
```

Comment:
Trivial; Perhaps we should use the same variable name as the query for consistency, and readability
```suggestion
export const getCacheKey = ( domain, mailboxName ) => [
```